book/website/community-handbook/local-build.md

```diff
@@ -0,0 +1,72 @@
+(ch-local-build)=
+# How to build Jupyter Book locally
```

Comment:
This title doesn't match the entry in the TOC. Should it?

Reply:
yes it should - amended in 2ca45fa43a1a2a2597b8387e641b2f6ab573334a